boilerplate/app/components/AutoImage.tsx

```diff
@@ -54,7 +54,7 @@ export function useAutoImage(
 /**
  * An Image component that automatically sizes a remote or data-uri image.
  *
- * - [Documentation and Examples](https://github.com/infinitered/ignite/blob/master/docs/Components-AutoImage.md)
+ * - [Documentation and Examples](https://github.com/infinitered/ignite/blob/master/docs/boilerplate/components/AutoImage.md)
```

Comment:
@frankcalise did we want to point these links over to docs.infinite.red?

Reply:
@morganick yeah good call, let's link to https://docs.infinite.red/ignite-cli/boilerplate/components/AutoImage/

Thanks for the contribution @ahinkle !